CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ cmake_minimum_required(VERSION 3.10.2 FATAL_ERROR)
 #============================================================================
 # Initialize the project
 #============================================================================
-project(ignition-common3 VERSION 3.6.1)
+project(ignition-common3 VERSION 3.6.2)
```

Comment:
Already mentioned to @mjcarroll , but for completeness I'll leave it here too, I think this should be a minor bump because of the new string join and env functions.

See comparison to 3.6.1:

https://github.com/ignitionrobotics/ign-common/compare/ignition-common3_3.6.1...ign-common3

Reply:
Changed to minor bump, thanks!